.ecp-gitlab-ci.yml

```diff
@@ -43,28 +39,15 @@ stages:
     - for i in ${BACKENDS}; do CMAKE_OPTS+=( -DCabana_REQUIRE_${i}=ON ); done
     - j="$(grep -c processor /proc/cpuinfo 2>/dev/null)" || j=0; ((j++))
     - mkdir build && cd build &&
-      cmake -DCMAKE_PREFIX_PATH=${CI_PROJECT_DIR}/kokkos.install -DCabana_ENABLE_TESTING=ON -DCabana_REQUIRE_SERIAL=OFF -DCabana_ENABLE_EXAMPLES=ON ${CMAKE_OPTS[@]} ${CMAKE_EXTRA_ARGS} .. &&
-      make -k -j${j} VERBOSE=1
-  tags:
-    - nobatch
-  artifacts:
-    paths:
-      - build/
-
-.Test:
-  stage: test
-  before_script:
-    - module load gcc
-  script:
-    - cd build
-    - make test CTEST_OUTPUT_ON_FAILURE=1
+      cmake -DCMAKE_PREFIX_PATH=${CI_PROJECT_DIR}/kokkos.install -DCabana_ENABLE_TESTING=ON -DCabana_ENABLE_SERIAL=OFF -DCabana_ENABLE_EXAMPLES=ON ${CMAKE_OPTS[@]} ${CMAKE_EXTRA_ARGS} .. &&
```

Comment:
`Cabana_ENABLE_SERIAL` is no longer an option. You can use `Cabana_REQUIRE_${device}` if you want to ensure a certain Kokkos backend is available.